README.md

```diff
@@ -579,6 +579,7 @@
 #### Image Management
 - [![Open-Source Software][OSS Icon]](https://github.com/KDE/digikam) [Digikam](https://www.digikam.org/) - DigiKam is an advanced digital photo management application for Linux.
 - [![Open-Source Software][OSS Icon]](https://github.com/szTheory/exifcleaner) [ExifCleaner](https://exifcleaner.com) - Remove image metadata with drag and drop. Supports multi-core batch processing.
+- [![Open-Source Software][OSS Icon]](https://github.com/jubalh/MAT) MAT - View exif data inside an image.
```

Comment:
I believe the tool is used for a wider use case. Metadata Anonymisation Toolkit, as the name says, helps improve the privacy posture of a file by anonymise the metadata. Thus I dont see this appropriate to be in this category. It could be in security category